### DISCUSSION TITLE
Ability to publish multiple coverage reports WITHOUT combining them?

### DISCUSSION THREAD

**Discussion by qwazwak** [danielpalme/ReportGenerator#449] [Q&A] [answered]
Is there any way to take multiple single code code reports, say opencover or cobertura, and for each create a Code coverage report/tab (for only that one report)? Specifically _without_ combining/merging reports together.

For context, this use case is in an Azure DevOps pipeline where we are generating a unit testing report, integration testing report, and a combined report.

**Comment by qwazwak** [comment]
Upon trying to get this, it seems this is an Azure issue where each time you publish code coverage, even if it was generated separately, Azure overwrites the previous report.

**Reply by danielpalme** [reply]
As far as I know you can only create one "Code coverage" tab.
You can generate several reports and save them as an artifact, but only one of them can be published in the "Code coverage" tab.

**Comment by qwazwak** [comment, marked as answer]
AFAIK , this is a limitation from the Azure DevOps side.
